extensions/pkg/webhook/certificates/reconciler.go

```diff
@@ -50,7 +50,7 @@ type reconciler struct {
 	// SyncPeriod is the frequency with which to reload the server cert. Defaults to 5m.
 	SyncPeriod time.Duration
 	// SeedWebhookConfig is the webhook configuration to reconcile in the Seed cluster.
-	SeedWebhookConfig *admissionregistrationv1.MutatingWebhookConfiguration
+	SeedWebhookConfig client.Object
```

Comment:
This is not really a `SeedWebhookConfig` anymore. Maybe it is a `SingleWebhookConfig` in comparison to the `ShootWebhookConfig` which might be something like `WebhookConfigWithNamespaceSelector`

Focussing on an users perspective there is [one log line](https://github.com/gardener/gardener/blob/e1b22b0936166dd52e10f34e156c40dab0a82dce/extensions/pkg/webhook/certificates/reconciler.go#L172) which might be confusing when it is running in an `operator` context because it includes the word `seed`.

Reply:
I renamed it to `SourceWebhookConfig`. However, making this even more generic is quite hard to achieve because at the least the shoot webhook config part is very specific to shoots. Anyways, I hope it's fine now,